<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>446</width>
                <height>319</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>446</width>
                <height>319</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>446</width>
                <height>319</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="label_task">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>6</y>
                        <width>351</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Segoe UI</family>
                        <pointsize>16</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Какое у вас образование:</string>
                </property>
            </widget>
            <widget class="QComboBox" name="comboBox">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>50</y>
                        <width>201</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Segoe UI</family>
                        <pointsize>10</pointsize>
                    </font>
                </property>
                <item>
                    <property name="text">
                        <string>Неполное среднее</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>Среднее</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>Среднее-специальное</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>Высшее</string>
                    </property>
                </item>
            </widget>
            <widget class="QLabel" name="label_img">
                <property name="geometry">
                    <rect>
                        <x>220</x>
                        <y>50</y>
                        <width>211</width>
                        <height>191</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QPushButton" name="btn_next">
                <property name="geometry">
                    <rect>
                        <x>231</x>
                        <y>260</y>
                        <width>120</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Segoe UI</family>
                    </font>
                </property>
                <property name="text">
                    <string>Вперёд</string>
                </property>
            </widget>
            <widget class="QPushButton" name="btn_back">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>260</y>
                        <width>120</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Segoe UI</family>
                    </font>
                </property>
                <property name="text">
                    <string>Назад</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_selected">
                <property name="geometry">
                    <rect>
                        <x>12</x>
                        <y>240</y>
                        <width>221</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Segoe UI</family>
                        <pointsize>10</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Выбрано:</string>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
